solr/modules/hadoop-auth/build.gradle

```diff
@@ -103,7 +103,7 @@ dependencies {
     exclude group: "org.apache.yetus", module: "audience-annotations"
   })
   // required for instantiating a Zookeeper server in tests or embedded
-  runtimeOnly ('org.xerial.snappy:snappy-java')
+  testRuntimeOnly ('org.xerial.snappy:snappy-java')
```

Comment:
this already exists above as `runtimeOnly`, but maybe we need it for tests as well? I don't know.

Reply:
I'd be surprised if the `runtimeOnly` version is needed?

at least in the `hdfs` module - there is only the `testRuntimeOnly`

Reply:
However in solr/core/build.gradle - `runtimeOnly ('org.xerial.snappy:snappy-java')` exists... so I'm conflicted.

Requires more digging but not going to stop this from getting merged.